test_runner/fixtures/neon_fixtures.py

```diff
@@ -2981,3 +2995,24 @@ def fork_at_current_lsn(
     """
     current_lsn = pg.safe_psql("SELECT pg_current_wal_lsn()")[0][0]
     return env.neon_cli.create_branch(new_branch_name, ancestor_branch_name, tenant_id, current_lsn)
+
+
+def list_inuse_ports():
```

Comment:
* `-> None`
* `print_ports_in_use`? Or `list_ports_in_use`. "in-use ports" sounds a little backward to me.